python/ray/autoscaler/_private/commands.py

```diff
@@ -90,10 +90,11 @@ def try_reload_log_state(provider_config: Dict[str, Any],
         return reload_log_state(log_state)
 
 
-def debug_status() -> str:
+def debug_status(status=None, error=None) -> str:
     """Return a debug string for the autoscaler."""
-    status = _internal_kv_get(DEBUG_AUTOSCALING_STATUS)
-    error = _internal_kv_get(DEBUG_AUTOSCALING_ERROR)
+    if status is None and error is None:
```

Comment:
Does this case ever happen? If not, can we just get rid of this code and make the args required?

Reply:
You're right, `commands.py#debug_status` is only ever called by the `ray status` CLI. Resolved now!